app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -170,8 +170,14 @@ export class Dispatcher {
 
     const repositoryIDs = localRepositories.map(r => r.id)
     await this.dispatchToSharedProcess<ReadonlyArray<number>>({ name: 'remove-repositories', repositoryIDs })
-
-    this.showFoldout({ type: FoldoutType.Repository })
+    await this.loadRepositories().then(response => {
```

Comment:
We shouldn't need to explicitly load repositories here. There are a few other ways we could accomplish this:

* `remove-repositories` already returns the removed IDs, so we could count how many repositories have have, minus the number removed.
* Change `remove-repositories` to return the list of repositories after removing the given ones.